src/csvToJson.js

```diff
@@ -17,27 +17,29 @@ let result = [];
 let folderName;
 let numberOfFiles;
 
-// @TODO change the name
-const fileWriter = (i, fileName, start, stop) => {
+const generateJsonFiles = (i, fileName, start, stop) => {
```

Comment:
this name is better, but it can be another place for confusion. because we have a generator module with methods inside.
but this name has more scores for me now

Reply:
Let me change that to something less confusing maybe

Reply:
ok